src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2561,6 +2635,21 @@ public void watchFiles(File buildFile, File outputDirectory, File testOutputDire
                     }
                 }
 
+                // check if resourceDirectory of an upstream project has been added
+                for (UpstreamProject p : upstreamProjects) {
+                    for (File resourceDir : p.getResourceDirs()) {
+                        if (!p.getResourceMap().get(resourceDir) && resourceDir.exists()) {
+                            registerAll(resourceDir.getCanonicalFile().toPath(), executor);
+                            p.getResourceMap().put(resourceDir, true);
+                        } else if (p.getResourceMap().get(resourceDir) && !resourceDir.exists()) {
+                            // deleted resource directory
+                            warn("The resource directory " + resourceDir
+                                    + " was deleted.  Restart liberty:dev mode for it to take effect.");
```

Comment:
In the message, just say "dev mode" instead of "liberty:dev mode".  Same for the section above (line 2633)